src/index.ts

```diff
@@ -245,14 +245,11 @@ export class Lrud {
 
     // ...remove all its children from the node ID list
     this.nodePathList = this.nodePathList.filter(nodeIdPath => {
-      return !(nodeIdPath.includes('.' + nodeId))
+      return !nodeIdPath.includes(path + '.children.')
     })
 
     // if the node we're unregistering was focusable, we need to remove it from
-    // our focusableNodePathList
-    this.focusableNodePathList = this.focusableNodePathList.filter(nodeIdPath => {
-      return !(nodeIdPath.includes('.' + nodeId))
-    })
+    this.focusableNodePathList.splice(this.focusableNodePathList.indexOf(path), 1)
```

Comment:
Is the `path` guaranteed to be in the `focusableNodePathList` at this point?

Reply:
Ooh, excellent catch. This should actually be wrapped in a check to ensure the node is focusable before we remove it from this array. Making the change now, 11/10 spot

Reply:
Actually, I think you've spotted some deeper broken logic around the `focusableNodePathList`. Will get some more tests added.

Reply:
Fixes pushed.